asmcli/commands/help.sh

```diff
@@ -264,6 +266,7 @@ FLAGS:
      --use_managed_cni
 
      --managed
+     --legacy
```

Comment:
We should also make sure that if `legacy` is passed without `managed` we fail installation and let the user know that it only makes sense in the managed context.

Reply:
Done.